### PR TITLE
fix warning, added subscription policy

### PR DESCRIPTION
**Pull request by hdJerry**
- fixed (Warning: CollectionCard: `key` is not a prop. Trying to access it will result in `undefined` being returned.)
- Added subscription policy

### CODE REVIEW

**Review by vyktoremario — DISMISSED** [review]

File: starters/shopify-next-tailwind/.env.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-STOREFRONT_API_SECRET=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-PUBLIC_STOREFRONT_API_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-PUBLIC_STOREFRONT_API_VERSION=2023-04
-PUBLIC_STORE_DOMAIN=xxxxxxxxxxx.myshopify.com
+SESSION_SECRET="op://Internal Shopify/env/prod/SESSION_SECRET"
+STOREFRONT_API_KEY="op://Internal Shopify/env/prod/STOREFRONT_API_KEY"
+STOREFRONT_API_SECRET="op://Internal Shopify/env/prod/STOREFRONT_API_SECRET"
+PUBLIC_STOREFRONT_API_TOKEN="op://Internal Shopify/env/prod/PUBLIC_STOREFRONT_API_TOKEN"
+PUBLIC_STOREFRONT_API_VERSION="op://Internal Shopify/env/prod/PUBLIC_STOREFRONT_API_VERSION"
+PUBLIC_STORE_DOMAIN="op://Internal Shopify/env/prod/PUBLIC_STORE_DOMAIN"
```
Comment (Megio) [inline starters/shopify-next-tailwind/.env.example:6]: Since we are working on starter.dev and this will be the env example file, I think we would like to revert this change and leave the file with `xxxx` characters.

**Review by Megio — APPROVED** [review]
